web/client/components/data/featuregrid/FeatureEditorFallback.jsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import ReactDock from 'react-dock';
+import { getDockSize } from '../../../selectors/featuregrid';
+import { mapLayoutValuesSelector } from '../../../selectors/maplayout';
+import BasicSpinner from '../../misc/spinners/BasicSpinner/BasicSpinner';
```

Comment:
Please use this component instead https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/misc/Loader.jsx

Reply:
Done